### ISSUE TITLE
Editor Problems

### ISSUE THREAD

**Issue by Aelbannan** [SiliconStudio/xenko#464]
![image](https://cloud.githubusercontent.com/assets/5529051/18657152/b2ca97b6-7eb4-11e6-9de6-cc6767527ec7.png)

I have 3 issues actually.

1. Scene view is in the wrong place
2. Studio is constantly taking a whole core, even when idle (I'm not doing anything).
3. All icons are super pixellated.

**Comment by Kryptos-FR** [comment]
Probably related to those issues (at least for point 1 and 3):
* https://github.com/SiliconStudio/xenko/issues/258
* https://github.com/SiliconStudio/xenko/issues/454

**Comment by Aelbannan** [comment]
Yeah seems like it. Any possible per-app workarounds? The global setting makes everything else too small.
Thanks for the quick reply.
